README.md

```diff
@@ -115,7 +115,7 @@ Vagrant.configure('2') do |config|
 end
 ```
 
-If you do not provide credentials, you will be prompted for them in the "up process."
+If you do not provide credentials, you will be prompted for them with maximum three attempts during the "vagrant up" process.
```

Comment:
@budhrg how about: You will receive a maximum of 3 prompts for the credentials, if you do not provide them during the ``Vagrant up`` process.